### PR TITLE
Remove extra spacing and divider below hero

### PR DESCRIPTION
**Pull request by hqhhuang**
Bug/issue #, if applicable: 89691884

## Summary
Remove extra spacing and divider below Hero, when there is no `Description` section or `PrimaryContent` Section

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] Added tests - NA
- [x] Ran `npm test`, and it succeeded 
- [ ] Updated documentation if necessary- NA

### CODE REVIEW

File: src/components/DocumentationTopic.vue [hunk 1]
```diff
@@ -292,6 +292,14 @@ export default {
         ? technologyList
         : [];
     },
+    showContainer: ({
+      isRequirement, deprecationSummary, downloadNotAvailableSummary,
+      primaryContentSections,
+    }) => (
+      isRequirement || (deprecationSummary && deprecationSummary)
```
Comment (marinaaisa) [inline src/components/DocumentationTopic.vue]:
```suggestion
      isRequirement
      || (deprecationSummary && deprecationSummary.length)
```

File: src/components/DocumentationTopic.vue [hunk 2]
```diff
@@ -29,7 +29,7 @@
           :platforms="platforms" :technologies="technologies"
         />
       </DocumentationHero>
-      <div class="container">
+      <div class="container" v-if="showContainer">
```
Comment (marinaaisa) [inline src/components/DocumentationTopic.vue]:
```suggestion
      <div v-if="showContainer" class="container">
```

File: src/components/DocumentationTopic.vue [hunk 3]
```diff
@@ -292,6 +292,14 @@ export default {
         ? technologyList
         : [];
     },
+    showContainer: ({
+      isRequirement, deprecationSummary, downloadNotAvailableSummary,
+      primaryContentSections,
```
Comment (marinaaisa) [inline src/components/DocumentationTopic.vue]:
```suggestion
      isRequirement,
      deprecationSummary,
      downloadNotAvailableSummary,
      primaryContentSections,
```

**Review by marinaaisa — APPROVED** [review]
I just added some styling suggestions :)
Good job, Hanqing! 👏

**Review by mportiz08 — APPROVED** [review]

**Pull request merged**